tests/fixtures/deduplicate_components_all_sections.json

```diff
@@ -0,0 +1 @@
+{"shared": {"components": {"schemas": {}}, "tags": []}, "/book": {"paths": {"/book": {"post": {"tags": ["book"], "summary": "Add a new book to the store", "description": "", "operationId": "addBook", "responses": {"405": {"description": "Invalid input"}}, "security": [{"bookstore_auth": ["write:books", "read:books"]}], "requestBody": {"$ref": "#/components/requestBodies/Book"}}}}, "components": {"schemas": {"Book": {"title": "a Book", "description": "A book for sale", "type": "object", "required": ["name", "photoUrls"], "properties": {"id": {"type": "integer", "format": "int64"}, "photoUrls": {"type": "array", "xml": {"name": "photoUrl", "wrapped": true}, "items": {"type": "string"}}, "status": {"type": "string", "description": "pet status in the store", "enum": ["available", "pending", "sold"]}}, "xml": {"name": "Book"}}}}, "tags": [{"name": "book", "description": "book information"}]}}
```

Comment:
can we format the JSON files so humans can read them? `json.dumps(json.loads(fp), indent=2, sort_keys=True)`

Reply:
Sure!

Reply:
I made them human-readable and pushed a new commit.